spec/app/ics-020-fungible-token-transfer/README.md

```diff
@@ -369,9 +526,32 @@ function onRecvPacket(packet: Packet) {
 function onAcknowledgePacket(
   packet: Packet,
   acknowledgement: bytes) {
-  // if the transfer failed, refund the tokens
-  if !(acknowledgement.success) {
-    refundTokens(packet)
+  // check if the packet that was sent is from a previously forwarded packet
+  prevPacket = privateStore.get(packetForwardPath(packet.sourcePort, packet.sourceChannel, packet.sequence))
+
+  if prevPacket != nil {
+    if acknowledgement.success {
```

Comment:
It' not really specified here or else where, as far as I can see, but is there any point in time where it is safe for the privateStore to remove these packets?

Reply:
Good point! Maybe it's possible to delete the `prevPacket` after `OnAcknowledgePacket` or `OnTimeoutPacket` have executed. What do you think, @AdityaSripal?

Reply:
Yes it is, added it in now